From: Jarek Poplawski <jarkao2@gmail.com>
Date: Mon, 1 Feb 2010 00:19:07 +0000
Subject: [PATCH 20/56] sky2: Fix TX_MAP_PAGE misspelling

commit 0c2dc318e88100e1abdff07e8a7d60460984a2d2 upstream.

Btw of the dma-debug problem reported by Michael Breuer I spotted
a tiny misspelling in TX_MAP_PAGE definition introduced by commit
6b84dacadbdc3.

Signed-off-by: Jarek Poplawski <jarkao2@gmail.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
---
 drivers/net/sky2.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/net/sky2.h b/drivers/net/sky2.h
index 365d79c..54cb303 100644
--- a/drivers/net/sky2.h
+++ b/drivers/net/sky2.h
@@ -2156,7 +2156,7 @@ struct tx_ring_info {
 	struct sk_buff	*skb;
 	unsigned long flags;
 #define TX_MAP_SINGLE   0x0001
-#define TX_MAP_PAGE     000002
+#define TX_MAP_PAGE     0x0002
 	DECLARE_PCI_UNMAP_ADDR(mapaddr);
 	DECLARE_PCI_UNMAP_LEN(maplen);
 };
-- 
1.7.1

